includes/Plugin.php

```diff
@@ -200,6 +200,12 @@ function ( $args ) {
 				return $args;
 			}
 		);
+
+		// WP CLI Commands.
+		if ( defined( 'WP_CLI' ) && WP_CLI ) {
+			$cli_commands = new \Google\Site_Kit\Core\CLI\CLI_Commands( $this->context );
+			$cli_commands->register();
```

Comment:
The `CLI_Commands` instance isn't significant, so let's register it similar to other classes where it isn't needed.
```suggestion
			( new \Google\Site_Kit\Core\CLI\CLI_Commands( $this->context ) )->register();
```

Reply:
Updated.